binding-library/java/src/main/java/com/microsoft/azure/functions/rabbitmq/annotation/RabbitMQOutput.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+ 
+package com.microsoft.azure.functions.rabbitmq.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.microsoft.azure.functions.annotation.CustomBinding;
+
+/**
+ * <p>Place this on a parameter whose value should be set in order for an event to be enqueued to a rabbitMQ queue.
+ * The parameter can be one of the following</p>
+ *
+ * <ul>
+ *     <li>A String java native type</li>
+ *     <li>Any POJO type</li>
```

Comment:
I don't think directly binding to byte[], int works for even dotnet even today. 

My understanding was that for in-proc, it would be string, POCO or BasicDeliverEventArgs and for out-of-proc, they would be normal strings or stringified json (except in java, json can be converted to the appropriate POJO as well)

Reply:
I'm only seeing converters for string-to-byte[] and for POCO-to-byte[] for output binding. How could other basic types be supported ?

Reply:
Here is how it is handled in Java: https://github.com/Azure/azure-functions-java-worker/blob/dev/src/main/java/com/microsoft/azure/functions/worker/binding/RpcIntegerDataSource.java 

For C#, this is the scenario I can think of if user adds a message to rabbitmq converting int to bytes : `byte[] intBytes = BitConverter.GetBytes(intValue);`  ,  trigger binding should support ` [RabbitMQTrigger("test_queue", ConnectionStringSetting = "rabbitMQ")] int res,` ?